mlflow/gateway/schemas/completions.py

```diff
@@ -0,0 +1,19 @@
+from typing import List
+
+from pydantic import BaseModel, Extra
+
+from .common import Metadata
+
+
+class Request(BaseModel, extra=Extra.allow):
```

Comment:
`Request` might be ambiguous. A name like `RequestBody` or `RequestPayload` might be better.

Reply:
`RequestPayload` acutally sounds good.